PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -2392,6 +2414,8 @@ function Get-SddcDiagnosticInfo
 
                     foreach ($cmd in $CmdsToLog) {
 
+                         "$(Get-Date -f o) Gathering $($cmd.C)" >> $transcript

```

Comment:
Show-Update etc. If you do like the 'o' format, I'd suggest doing that in Show-Error/Warning/Update so its consistent across the module. I suppose having the GMT offset is useful.